mcs/tools/mdoc/Test/html.expected/Mono.DocTest.Generic/Extensions.html

```diff
@@ -363,22 +363,22 @@ <h4 class="Subsection">Parameters</h4>
             </dl>
           </blockquote>
           <h2 class="Section">Remarks</h2>
-          <div class="SectionBox" id="M:Mono.DocTest.Generic.Extensions.ForEach``1(System.Collections.Generic.IEnumerable{``0},System.Action{``0}):Remarks">
+          <div class="SectionBox" id="M:Mono.DocTest.Generic.Extensions.ForEach{T}(System.Collections.Generic.IEnumerable{T},System.Action{T}):Remarks">
             <tt>M:Mono.DocTest.Generic.Extensions.ForEach``1</tt>
           </div>
           <h2 class="Section">Requirements</h2>
-          <div class="SectionBox" id="M:Mono.DocTest.Generic.Extensions.ForEach``1(System.Collections.Generic.IEnumerable{``0},System.Action{``0}):Version Information">
+          <div class="SectionBox" id="M:Mono.DocTest.Generic.Extensions.ForEach{T}(System.Collections.Generic.IEnumerable{T},System.Action{T}):Version Information">
             <b>Namespace: </b>Mono.DocTest.Generic<br /><b>Assembly: </b>DocTest (in DocTest.dll)<br /><b>Assembly Versions: </b>0.0.0.0</div>
           <hr size="1" />
         </blockquote>
         <h3 id="M:Mono.DocTest.Generic.Extensions.ToDouble(System.Collections.Generic.IEnumerable{System.Int32})">ToDouble Method</h3>
         <blockquote id="M:Mono.DocTest.Generic.Extensions.ToDouble(System.Collections.Generic.IEnumerable{System.Int32}):member">
           <div class="msummary">
-            <a href="http://www.go-mono.com/docs/monodoc.ashx?link=T:System.Collections.Generic.IEnumerable`1">IEnumerable&lt;int&gt;</a> 
+            <a href="http://www.go-mono.com/docs/monodoc.ashx?link=T:System.Collections.Generic.IEnumerable&lt;System.Int32&gt;">IEnumerable&lt;int&gt;</a> 
```

Comment:
What's additionally odd is that a link for `IEnumerable<int>` -- which *was* to the [`IEnumerable<T>` documentation](http://docs.go-mono.com/index.aspx?link=T:System.Collections.Generic.IEnumerable%601) -- has replaced `IEnumerable^1` in the URL with `IEnumerable<System.Int32>` in the URL.

(Using `^` to represent the backtick, because I can't figure out how to use it otherwise.)

This is *wrong*, for at least two reasons:

1. [The new url *doesn't work*](http://www.go-mono.com/docs/monodoc.ashx?link=T:System.Collections.Generic.IEnumerable&lt;System.Int32&gt;):

    > *Error:* Your request has found no candidate provider [hs="(null)", id="(null)"]

2. *Conceptually*, it's wrong. The docs for `IEnumerable<int>` *do not* differ from `IEnumerable<char>` or `IEnumerable<byte>`, because .NET doesn't support partial template specialization (or anything like it). *There is only one generic type*; there is thus *no reason* to pass along the type parameters within the URL.

The *current* output is correct; the new output is not.